AstSemantics.md

```diff
@@ -154,6 +151,28 @@ by [available resources](Nondeterminism.md)). In the future, to support
 [>4GiB linear memory](FutureFeatures.md#heaps-bigger-than-4gib), support for
 indices with type `int64` will be added.
 
+### Adjusting memory size and permissions: `mmap`
+
+In the MVP the size of linear memory is fixed: The initial size of linear
+memory will remain unchanged for the life of that WebAssembly module. Later,
+we will support a limited form of `mmap` which can:
+
+  * Allocate pages of memory, in order to increase or decrease
+    the amount of available memory to the WebAssembly module.
+  * Adjust the permissions of a page of memory, for example to make
+    small effective addresses close to `0` behave as if they are
+    [out-of-bounds](AstSemantics.md#out-of-bounds)
+    (see [discussion](https://github.com/WebAssembly/design/issues/204)).
+
+In addition to the `mmap` operation, applications will also have access to
+
+ * `munmap`, to unmap `mmap`ed pages, and
+ * `mmap_get_page_size`, to detect the proper system page size.
```

Comment:
@jfbastien ok, I pushed an edit about the "good enough" aspect.

@pizlonator I might be confused between email and github's interface, but I think @jfbastien was commenting on the pull here, not responding to your question.

Regarding large reservations, I don't think those would be necessary (unless I am missing something?), but page-level protections would be (but they already were necessary in the docs, as they mentioned a way to protect the area around 0).

I also wrote a general response in the main thread of this pull.